agt/state.py

```diff
@@ -44,17 +54,20 @@ def __init__(self, output_callback):
 
         self.memory = {}
 
-    async def say(self, text: str, image_url: str = None):
+    async def say(self, text: str, image_url: str = None, ssml: str = ""):
```

Comment:
```suggestion
    async def say(self, text: str, image_url: Optional[str] = None, ssml: Optional[str] = None):
```
None so we know when there is no ssml and we can fill it from the text field